src/js/tracing/reactnativetracing.ts

```diff
@@ -350,6 +360,17 @@ export class ReactNativeTracing implements Integration {
     return this._inflightInteractionTransaction;
   }
 
+  /**
+   *  Sets the current view name into the app context.
+   *  @param event Le event.
+   */
+  private _getCurrentViewEventProcessor(event: Event): Event {
+    if (event.contexts && this._currentViewName) {
+      event.contexts.app = { ...{ view_names: [this._currentViewName] }, ...event.contexts.app };
```

Comment:
We can avoid the creation of the object wrapping `view_names`.

```suggestion
      event.contexts.app = { view_names: [this._currentViewName], ...event.contexts.app };
```